iam/snippets/member_add.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START iam_modify_policy_add_member]
+import (
+	"fmt"
+	"io"
+
+	"google.golang.org/api/iam/v1"
```

Comment:
PTAL at IAM instructions (b/281691389#comment2) as we should be using latest Client Libraries for [IAM in GoLang](https://github.com/googleapis/google-cloud-go/tree/main/iam)

Reply:
v1 is actually correct here because the sample is for allow policy. v2 is only used for deny policy.